### DISCUSSION TITLE
How to solve Aborted (core dumped)

### DISCUSSION THREAD

**Discussion by SiraHaruethaipree** [abetlen/llama-cpp-python#465] [Q&A]
llama.cpp: loading model from ./model/ggml_model_q4_0.bin
terminate called after throwing an instance of 'std::runtime_error'
  what():  unexpectedly reached end of file
Aborted (core dumped)
![image](https://github.com/abetlen/llama-cpp-python/assets/82432680/fcd914dd-c5af-4a11-9553-49e9dfed02f5)

**Comment by gjmulder** [comment]
That is a C++ error likely from `llama.cpp`. Does the same model work with `llama.cpp`?

**Reply by SiraHaruethaipree** [reply]
I try to solve with this solution as change the version of library as indicate in link below. but it still error.
https://github.com/oobabooga/text-generation-webui/discussions/1615#discussioncomment-6021733
when I try to load another model with latest version llama-cpp-python=0.1.70. it can work. as show image below
![image](https://github.com/abetlen/llama-cpp-python/assets/82432680/7a34dc51-1856-4003-be10-80808d85a7b9)

**Reply by nthomsencph** [reply]
Same issue
